tensorflow/core/platform/env.h

```diff
@@ -373,6 +377,11 @@ struct ThreadOptions {
   size_t guard_size = 0;  // 0: use system default value
 };
 
+/// A utility routine: copy contents of `src` in file system `src_fs`
+/// to `target` in file system `target_fs`.
+Status CopyFile(FileSystem* src_fs, const string& src, FileSystem* target_fs,
```

Comment:
It might confuse users if env.h provides two APIs for CopyFile. The CMake error also points to this helper being the linker error. I'm not entirely sure why, but if the helper is removed it might fix the problem.

Reply:
Thanks @jart for the help and sorry for the late reply. I made a change to the name `CopyFile` -> `FileSystemCopyFile` and it seems that Windows build is successful. Please take a look.